.gitignore

```diff
@@ -14,3 +14,4 @@ README.html
 es-tmp/
 clouddriver-oracle-bmcs/bmcs-sdk
 gatling.conf
+*.DS_Store
```

Comment:
don't mind having it here, but you probably will want to add this to your global gitignore at `~/.gitignore`

Reply:
Actually please remove it as we wont be able to merge that upstream, and update your global gitignore

Reply:
done.